pkgs/development/python-modules/keras/default.nix

```diff
@@ -1,39 +1,73 @@
-{ lib, buildPythonPackage, fetchPypi
-, pytest, pytest-cov, pytest-xdist
-, six, numpy, scipy, pyyaml, h5py, optree
-, keras-applications, keras-preprocessing
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, setuptools
+, absl-py
+, dm-tree
+, markdown-it-py
+, ml-dtypes
+, namex
+, numpy
+, optree
+, rich
+, tensorflow
+
+
+, fetchPypi
+, pytest
+, pytest-cov
+, pytest-xdist
+, six
+, scipy
+, pyyaml
+, h5py
+, keras-applications
+, keras-preprocessing
```

Comment:
a number of these arguments are not used

Reply:
Indeed, I removed them.